Gemfile

```diff
@@ -5,6 +5,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.6'
 
+gem 'activerecord-import'
```

Comment:
@weedySeaDragon Just curious, is there a reason why we are not using [`upsert_all`](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert_all) and instead relying on this gem?

Reply:
Ah.  excellent question.  Just habit. 
I'll play with `upsert_all` today and see if it's cleaner.

Reply:
So I ran some simple tests.  The performance is pretty much the same between using the `activerecord-import` gem and using `insert_all!`.  
When using `insert_all!` I do have to manually create the array of Hashes for each row that needs to be inserted. (take the column headers and make them the keys for all of the values in a csv row).  `activerecord-import` takes care of that.  
Seems at this point, `activerecord-import` is fine to use with relatively small sizes (thousands),and it does provide a fair amount of validity checks w.r.t. the column names (checking that they work with the attribute names), etc.  We're not using any of that functionality, of course, so it's of no benefit.
(And in the repo for the `activerecord-import` gem, people are definitely asking what the future of it is given `insert_all` and `upsert..` now in Rails 6.)

I'll remove the gem and just use `insert_all!`.  That's 1 less gem that is needed.

Good catch!